src/Compilers/VisualBasic/Test/Syntax/Parser/VisualBasicParseOptionsTests.vb

```diff
@@ -119,7 +119,7 @@ Public Class VisualBasicParseOptionsTests
             GetValues(GetType(LanguageVersion)).
             Cast(Of LanguageVersion).
             Select(Function(x) CInt(x)).
-            Where(Function(x) x <> LanguageVersion.Latest).
+            Where(Function(x) x < LanguageVersion.Latest).
```

Comment:
>Where(Function(x) x < LanguageVersion.Latest). [](start = 12, length = 46)

There are no changes to the LanguageVersion enum, what is the reason to change this condition?